<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>stcFreqDialog</class>
 <widget class="QDialog" name="stcFreqDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frequency analysis</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QListWidget" name="listWidgetStcs"/>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="fminLabel">
       <property name="text">
        <string>fmin</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxFmin">
       <property name="suffix">
        <string>Hz</string>
       </property>
       <property name="value">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="fmaxLabel">
       <property name="text">
        <string>fmax</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxFmax">
       <property name="suffix">
        <string>Hz</string>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="freqIntervalLabel">
       <property name="text">
        <string>freq interval</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxInterval">
       <property name="minimum">
        <double>1.000000000000000</double>
       </property>
       <property name="value">
        <double>3.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="tminLabel">
       <property name="text">
        <string>tmin</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTmin">
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="minimum">
        <double>-2.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="tmaxLabel">
       <property name="text">
        <string>tmax</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTmax">
       <property name="suffix">
        <string>s</string>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>0.500000000000000</double>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="hemisphereLabel">
       <property name="text">
        <string>hemisphere</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxHemisphere">
       <item>
        <property name="text">
         <string>left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>right</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="njobsLabel">
       <property name="text">
        <string>njobs</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QSpinBox" name="spinBoxNjobs">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QRadioButton" name="radioButtonFixed">
         <property name="text">
          <string>Fixed</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ncyclesLabel">
         <property name="text">
          <string>ncycles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxNcycles">
         <property name="minimum">
          <double>1.000000000000000</double>
         </property>
         <property name="maximum">
          <double>100.000000000000000</double>
         </property>
         <property name="value">
          <double>7.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QRadioButton" name="radioButtonAdapted">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can define different number of cycles for each frequency of interest. The defined factor is multiplied by the frequency in hertz to get the number of cycles (ncycles = freq * factor). For instance, with a factor of 0.5, frequencies of 2, 4 and 6 tranlslate to morlet wavelets with cycles of 1, 2 and 3.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Adapted</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can define different number of cycles for each frequency of interest. The defined factor is multiplied by the frequency in hertz to get the number of cycles (ncycles = freq * factor). For instance, with a factor of 0.5, frequencies of 2, 4 and 6 tranlslate to morlet wavelets with cycles of 1, 2 and 3.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>cycles / freq</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBoxCycleFactor">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can define different number of cycles for each frequency of interest. The defined factor is multiplied by the frequency in hertz to get the number of cycles (ncycles = freq * factor). For instance, with a factor of 0.5, frequencies of 2, 4 and 6 tranlslate to morlet wavelets with cycles of 1, 2 and 3.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="minimum">
          <double>0.010000000000000</double>
         </property>
         <property name="maximum">
          <double>2.000000000000000</double>
         </property>
         <property name="singleStep">
          <double>0.100000000000000</double>
         </property>
         <property name="value">
          <double>0.500000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Clicking 'OK' will plot the average TFR over all epoch 
source estimates. Click on the desired frequency 
in the plot to show the power on the surfaces.</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidgetStcs</tabstop>
  <tabstop>doubleSpinBoxFmin</tabstop>
  <tabstop>doubleSpinBoxFmax</tabstop>
  <tabstop>doubleSpinBoxInterval</tabstop>
  <tabstop>doubleSpinBoxTmin</tabstop>
  <tabstop>doubleSpinBoxTmax</tabstop>
  <tabstop>comboBoxHemisphere</tabstop>
  <tabstop>spinBoxNjobs</tabstop>
  <tabstop>radioButtonFixed</tabstop>
  <tabstop>doubleSpinBoxNcycles</tabstop>
  <tabstop>radioButtonAdapted</tabstop>
  <tabstop>doubleSpinBoxCycleFactor</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>stcFreqDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>stcFreqDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>571</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>ncyclesLabel</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>449</x>
     <y>295</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>298</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxNcycles</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>436</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>607</x>
     <y>292</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonAdapted</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>422</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>531</x>
     <y>331</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonAdapted</sender>
   <signal>toggled(bool)</signal>
   <receiver>doubleSpinBoxCycleFactor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>331</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>329</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
